cdserver/config/initializers/new_framework_defaults.rb

```diff
@@ -9,7 +9,7 @@
 ActiveSupport.to_time_preserves_timezone = true
 
 # Require `belongs_to` associations by default. Previous versions had false.
-Rails.application.config.active_record.belongs_to_required_by_default = true
+Rails.application.config.active_record.belongs_to_required_by_default = false
 
```

Comment:
This is a new feature turned on by default in Rails 5. I turned it off to make it easier to play around with models without creating all associations. Perhaps it will make sense to turn it back on later.